### ISSUE TITLE
How does the loss function train the network?

### ISSUE THREAD

**Issue by wysot** [yihui-he/softer-NMS#17]
Hello, how do the two Lcls and Lreg loss functions in the paper train the network? Are they separate training networks? Or do two functions turn into one to train the network?

**Comment by yihui-he** [comment]
It's the same as [faster R-CNN](https://arxiv.org/abs/1506.01497). Please refer to it for details.
BTW, this repo is deprecated. Please use our new repo: https://github.com/yihui-he/KL-Loss
